frontend/lib/src/components/shared/InputInstructions/InputInstructions.tsx

```diff
@@ -54,7 +54,7 @@ const InputInstructions = ({
   }
 
   // Show enter instruction if not a form or form allows Enter to submit
-  if (dirty && allowEnterToSubmit) {
+  if (allowEnterToSubmit) {
```

Comment:
@mayagbarnes  I ended up keeping only `allowEnterToSubmit` here. I figured this being a shared util component it could rely on parents tell it to render or not (e.g. it shouldn't care about the `dirty` state)

Can potentially do the same for max length below as well separately